docs/installation-configuration.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Unifi Setup
+meta_desc: Information on how to install the Unifi provider.
+layout: installation
+---
+
+## Installation
+
+The Pulumiverse Unifi provider is available as a package in all Pulumi languages:
```

Comment:
Not entirely true. YAML is not yet supported. Maybe something like `as a package for the following Pulumi languages:` Or we ignore this minor issue and add YAML support as quickly as possible.
Other than that, Java is missing in the list (same for the README file)

Reply:
YAML is supported as no SDK need to be generated. You only need to configure the provider in your `Pulumi.yaml` file and you are good to go. See the docs here:

https://www.pulumi.com/docs/reference/yaml/#providers-and-provider-versions

Java support is indeed missing as you can't immediately push to the main package registry. To be investigated.